rules/composer/tests/Modifier/ComposerModifierTest.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace Rector\Composer\Tests\Modifier;
+
+use Nette\Utils\Json;
+use PHPUnit\Framework\TestCase;
+use Rector\Composer\ValueObject\AddPackage;
+use Rector\Composer\ValueObject\ChangePackage;
+use Rector\Composer\ValueObject\ChangePackageVersion;
+use Rector\Composer\ValueObject\MovePackage;
+use Rector\Composer\ValueObject\RemovePackage;
+use Rector\Composer\Modifier\ComposerModifier;
+
+final class ComposerModifierTest extends TestCase
+{
+    public function testRefactorWithOneAddedPackage(): void
+    {
+        $composerModifier = new ComposerModifier();
```

Comment:
This one should be service and used with `AbstractKernelTestCase`. See other tests how to get a service from container

Reply:
Ok I will try to rewrite it